### ISSUE TITLE
Movie Mode for H6099

### ISSUE THREAD

**Issue by ding-ma** [wez/govee2mqtt#334]
### Govee Device SKU

H6099

### Govee2MQTT Version

c9d27764095fabb1d1ee339ab187fc840cf00c83

### Describe the issue

The movie mode of H6099 should use the onboard camera to change the color of the lights based off of the screen. 
This works when triggered through the govee app but not through the effect -> movie in the light setting in hass.

### Startup Diagnostics

docker logs -->

```
[2024-12-11T23:08:34 INFO  govee::service::state] requesting update via Platform API TV Backlight 3 Lite (14:B7:D4:0F:44:C6:77:6D H6099) Some(DeviceState { on: true, light_on: Some(true), online: Some(true), kelvin: 2000, color: DeviceColor { r: 0, g: 0, b: 0 }, brightness: 100, scene: None, source: "PLATFORM API", updated: 2024-12-12T06:57:35.434040889Z })
[2024-12-11T23:08:42 INFO  govee::service::hass] Command for TV Backlight 3 Lite (14:B7:D4:0F:44:C6:77:6D H6099): {"state":"ON","effect":"Movie"}
[2024-12-11T23:08:42 INFO  govee::service::state] Using Platform API to set TV Backlight 3 Lite (14:B7:D4:0F:44:C6:77:6D H6099) to scene Movie
[2024-12-11T23:08:43 INFO  govee::platform_api] control_device result: ControlDeviceResponse { request_id: "uuid", code: 200, message: "success", capability: ControlDeviceResponseCapability { kind: DynamicScene, instance: "lightScene", value: Object {"id": Number(168), "paramId": Number(150)}, state: Object {"status": String("success")} } }
```

**Comment by nyr3188** [comment]
I have the same device and am experiencing the same issue. HA can set any of the scenes except for Game or Movie which are the two that use the camera to change the colors of the lights. When you tap on Video in the Govee app, it makes you click a second button to "Start to Experience" before it actually starts using the camera.
![Screenshot_20250104-172205](https://github.com/user-attachments/assets/22a15195-7d23-4213-80fc-8d6e52c0c0de)

**Comment by Thorsten2605** [comment]
Is it possible to use this integration for model H6099?
I thought that LAN API isn't available for this device...
How could you enable this for this model?

**Comment by nyr3188** [comment]
You're right that it doesn't seem to have the LAN API option available but it's showing up and is controllable except for the Video option as is the subject of this thread

![Screenshot_20250112-005834](https://github.com/user-attachments/assets/455ed110-4acf-40d0-a6f8-9ef25e7c40a9)

**Comment by nyr3188** [comment]
@ding-ma For what it's worth, I found a workaround that I've tested and it seems to work.

1. While your H6099 is in Movie mode and it's active, create a Snapshot in the Govee Home App by tapping the floating green button.
2. Give the snapshot a unique name and then save it.
3. Restart Govee to MQTT Add-On
4. The new Snapshot you just created should show up in the Effects list in Home Assistant and it should automatically activate the camera and start changing your lights based on what's on your screen.

**Comment by Phunky** [comment]
Hmmm mine shows up but I can't control the light at all

**Comment by ding-ma** [comment]
we found a workaround through the google voice assistant and alex api. When the tv turns on, we tell HA to send a voice prompt to google voice assistant to set the backlight mode to "movie" and it works 90% of the time

**Comment by eXPerience83** [comment]
> we found a workaround through the google voice assistant and alex api. When the tv turns on, we tell HA to send a voice prompt to google voice assistant to set the backlight mode to "movie" and it works 90% of the time

And what is the exact command you say to Google Assistant?

**Comment by eXPerience83** [comment]
OK, you can turn on "Movie" Effect for H6099 (TV Backlight 3 Lite) and it's like Dreamview but for only this device, It doesn't appear "Dreamview" button for H6099.

For H6640 (RGBIC Neon Rope Lights) there is a button for enable "DreamView" or you can enable "Movie" effect, but it doesn't work as H6099 and the DreamView option doesn't do anything.

Solution: create 2 scenes on the official Govee app, one to enable Dreamview (it will turn on every lights connected to DreamView settings) and the other to turn off DreamView and after that turn off all the lights.

For now, I'm not using the Google Assistant integration. This work for HA OS.

Thank you very much!!

**Comment by dennisvandehoef** [comment]
@enbacode did you try the workaround posted by @nyr3188 in this issue? that one works stable for 8 months or so for me.
